java/src/Ice/src/main/java/com/zeroc/Ice/ConnectionOptions.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) ZeroC, Inc.
```

Comment:
Is this the copyright style we're using going forward?
Everywhere else in Java (and C++) it seems we're using:
`// Copyright (c) ZeroC, Inc. All rights reserved.`

Reply:
Yes, we should update all the files to use the simpler copyright notice.